components/organisms/messaging/components/base/components/menuIcon.js

```diff
@@ -0,0 +1,12 @@
+export const MenuIcon = props => {
+  return (
+    <svg width='32' height='32' viewBox='0 0 32 32' fill='none' xmlns='http://www.w3.org/2000/svg'>
+      <path
+        fillRule='evenodd'
+        clipRule='evenodd'
+        d='M16 12C17.1 12 18 11.1 18 10C18 8.9 17.1 8 16 8C14.9 8 14 8.9 14 10C14 11.1 14.9 12 16 12ZM16 14C14.9 14 14 14.9 14 16C14 17.1 14.9 18 16 18C17.1 18 18 17.1 18 16C18 14.9 17.1 14 16 14ZM14 22C14 20.9 14.9 20 16 20C17.1 20 18 20.9 18 22C18 23.1 17.1 24 16 24C14.9 24 14 23.1 14 22Z'
+        fill='#999999'
+      />
+    </svg>
+  )
+}
```

Comment:
Why are we storing this file (and others) as `.js` and not `.svg`?

Reply:
i think this way (looking at an svg file as a component) will give us more control over properties